### PR TITLE
Remove duplicate exercises and add ch2 folder

### PR DESCRIPTION
**Pull request by bcrey2**
Time for you to sign the treaty and end this pointless debate in the senate.

image

### CODE REVIEW

File: chapter-2-the-integers/exercice-17a.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Let $f_n$ be the $n$ th fibbonacci number, with $f_1 = 1$ and $f_2 = 1$ Prove $f_n < 2^n$ for $n \in \mathbb{N}$
+
+**Base Cases**
+
+Let $n = 1$. then
```
Comment (bcrey2) [inline chapter-2-the-integers/exercice-17a.md:5]: As you wish, my master.

image

File: chapter-2-the-integers/exercice-17a.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Using the inductive hypothesis
+
+&emsp; $f_{k+1} < 2^k + 2^{k-1}$
+
+&emsp; $f_{k+1} < 2^k + 2^k$
+
+&emsp; $f_{k+1} < 2 \cdot 2^{k}$
+
+&emsp; $f_{k+1} < 2^{k + 1}$
```
Comment (astute-teletubby) [inline chapter-2-the-integers/exercice-17a.md:29]: We-san gonna speed you somewhere.

image

File: all-exercises/exercises.md
Comment (bcrey2) [inline all-exercises/exercises.md:1]: We-san gonna speed you somewhere.

image

**Review by cryptic-spinach — APPROVED** [review]
The queen doesn't need to know.

image

**Pull request merged**